src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Date.cs

```diff
@@ -121,7 +121,7 @@ public static bool TryFormat(DateTimeOffset value, Span<byte> buffer, out int by
                     return TryFormatDateTimeG(value.DateTime, offset, buffer, out bytesWritten);
 
                 default:
-                    throw new FormatException(SR.Argument_BadFormatSpecifier);
+                    return ThrowHelper.TryFormatThrowFormatException(out bytesWritten);
```

Comment:
nit: assign bytesWritten to default outside the switch statement so that we don't have to have an out parameter on the ThrowHelper. It looks strange that it is returning an out parameter.

Reply:
We'd be writing `bytesWritten` twice and we'd still need a strange looking return type on the helper to avoid ruining the 100% code coverage on Utf8Formatter. ThrowHelper is always going to look strange unless you're one the people deep into JIT-fu. It still looks strange to me.